src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -260,6 +297,8 @@ const AddAppealForm = (): ReactElement => {
                           variant="outlined"
```

Comment:
For these 4 text inputs could we make them stay inline until the screen size reaches xs? You could just change all the md={} to sm={} on the Grids.